network-protection/network-protection-impl/src/main/res/layout/activity_netp_notification_settings.xml

```diff
@@ -54,6 +54,18 @@
                 android:layout_height="wrap_content"
                 android:orientation="vertical"
         >
+
+            <com.duckduckgo.mobile.android.ui.notifyme.NotifyMeView
+                    android:id="@+id/vpn_notification_settings_notify_me"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:visibility="gone"
+                    app:primaryText="Get notified of VPN events"
```

Comment:
NIT: Why not have this in donotranslate?

Reply:
will do, missed it